modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -66,3 +66,32 @@ Defaults to 4 seconds.
 The presence ping gather job parameter changes the timeout setting for gathering the presence ping, in seconds.
 Adjust the [systemitem]``java.salt_presence_ping_gather_job_timeout`` parameter.
 Defaults to 1 second.
+
+
+== Disabling the Salt Mine
+
+In older versions, {productname} used a tool called Salt mine to check minion availability.
+Salt mine would ping the server every hour, which created significant load.
```

Comment:
Hmm this is not actually accurate. What happens actually is, when the Salt mine is enabled on the minions, an event is triggered every hour from the minion and is processed by the Server, and we use that as a "checkIn".

Maybe we could rephrase to something like "With the Salt mine enabled, the minions would checkIn every hour".